src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -621,7 +606,7 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
         /// <summary>
         /// Performs required static initialization in the Edge context.
         /// </summary>
-        private static void Initialize()
+        private static async Task Initialize()
```

Comment:
Could, but many scenarios I prefer awaiting for a few reasons (including clarity, ease of debug, etc., as there will be a difference). I'll dig up some good information I have on this and send it your way.